### PR TITLE
Updating (and fixing spelling) of SurveyResponseDraft to utilize...

### PR DESCRIPTION
**Pull request by eriklundberg**
ForeignKeys

### CODE REVIEW

File: apps/survey/views.py [hunk 1]
```diff
@@ -312,30 +312,43 @@ def show_survey(request, survey_short_name):
...
         prefilled_data = {}
 
     # we inject the birthyear that was provided with the idcode so we know how old
     # the user is. There must be a question with this name in the survey for this to be usable
     # tipically it would be set to hidden.
-    prefilled_data['PREFIL_BIRTHYEAR'] = IdCodeObject.fodelsedatum
+    prefilled_data['PREFIL_BIRTHYEAR'] = id_code.fodelsedatum
 
     form = survey.as_form()(prefilled_data)
-    #survey.set_form(form)
```
Comment (hrpt-se) [inline apps/survey/views.py]: Why is this removed?

Comment (eriklundberg) [reply apps/survey/views.py]: Commented line of code without explanation in the middle of the code base. Removed since it doesn't have any purpose.
